lib/stellar/client.rb

```diff
@@ -3,7 +3,7 @@
 require 'securerandom'
 
 module Stellar
-  class InvalidSep10ChallengeError < StandardError; end
+  class AccountRequiresMemoError < StandardError; end
```

Comment:
@JakeUrban can you add the accountId and the operationIndex similar to https://github.com/stellar/js-stellar-sdk/blob/master/src/errors.ts#L90?